kapten/server.py

```diff
@@ -64,6 +64,57 @@
     )
 
 
+@app.route("/webhook/github", methods=["POST"])
+async def github_webhook(request):
+    logger.info("Received GitHub webhook from: %s", request.client.host)
+
+    # Validate event type
+    event_type = request.headers.get("x-github-event") or ""
+    if event_type.lower() != "deployment":
+        logger.debug("Responding to unwanted GitHub event: {}".format(event_type))
+        return Response("Event not handled by Kapten", status_code=202)
```

Comment:
Maybe answer with `404` here as well, to not reveal existence of the route when header is missing.

```suggestion
        return Response("Event not handled by Kapten", status_code=404)
```

Reply:
We could, although we'd have to also filter out the `"ping"` event and respond properly to that.